src/ansi.rs

```diff
@@ -0,0 +1,1941 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// This module was originally part of the `alacritty_terminal` crate, which is
+// licensed under the Apache License, Version 2.0 and is part of the Alacritty
+// project (https://github.com/alacritty/alacritty).
```

Comment:
I don't think this is necessary considering both repositories are part of the same project and both are licensed under MIT+Apache.

In its current state this is more harmful than helpful, since using it as Apache only would not technically allow usage of VTE under MIT anymore. And I don't think it's worth specifying the multi-license.